.pre-commit-config.yaml

```diff
@@ -79,14 +79,22 @@ repos:
         additional_dependencies:
           - jinja2
 
-      - id: render-readme-versions
-        name: Update versions in README.md to match their defaults values
+      - id: propagate-ansible-variables
+        name: Update static files referencing default kubespray values
         language: python
         additional_dependencies:
           - ansible-core>=2.16.4
-        entry: scripts/render_readme_version.yml
+        entry: scripts/propagate_ansible_variables.yml
         pass_filenames: false
 
+      - id: check-checksums-sorted
+        name: Check that our checksums are correctly sorted by version
+        entry: scripts/assert-sorted-checksums.yml
+        language: python
+        pass_filenames: false
+        additional_dependencies:
+          - ansible
```

Comment:
I see their (`propagate_ansible_variables.yml` and `assert-sorted-checksums.yml`) shebang are same. (`#!/usr/bin/env ansible-playbook`)

Do we have to write `ansible`? Or could we use `ansible-core` like above?

Reply:
Nope, assert-sorted rely on the `community.general` collection, which is only included in the `ansible` package, not in `ansible-core`

Reply:
Got it. Thank you!

/lgtm